development-docs/systemtests/io.strimzi.systemtest.operators.user.UserST.md

```diff
@@ -0,0 +1,136 @@
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Deploy Kafka cluster with TLS authentication and Simple authorization. | Kafka cluster is deployed with TLS listener and Simple ACL authorization enabled. |
+| 2. | Create TLS external user with ACL permissions. | TLS external user is created with specified ACL rules for topic access. |
+| 3. | Create custom external TLS secret for user. | External TLS secret is created with custom certificates. |
+| 4. | Test message sending and receiving with external TLS user. | Messages are successfully sent and received using external TLS certificates. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+
+## testTlsExternalUserWithQuotas
+
+**Description:** Verifies that TLS external authenticated Kafka users can be configured with quotas.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create TLS external user with quota configuration. | User is created successfully with TLS external authentication and quota settings applied. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+
+## testTlsUserWithQuotas
+
+**Description:** Verifies that TLS authenticated Kafka users can be configured with quotas.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create TLS user with quota configuration. | User is created successfully with TLS authentication and quota settings applied. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+
+## testUpdateUser
+
+**Description:** Verifies updating a Kafka user from TLS to SCRAM-SHA authentication and validates user secret contents.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create TLS Kafka user. | User is created with TLS authentication and secret contains TLS certificates. |
+| 2. | Verify TLS user secret contents. | Secret contains ca.crt, user.crt, and user.key fields. |
+| 3. | Test message sending and receiving with TLS user. | Messages are successfully sent and received. |
+| 4. | Update user authentication to SCRAM-SHA-512. | User authentication is updated successfully. |
+| 5. | Verify SCRAM user secret contents. | Secret contains SCRAM password field and TLS certificates are removed. |
+| 6. | Test message sending and receiving with SCRAM user. | Messages are successfully sent and received with SCRAM authentication. |
+
+**Labels:**
+
+* `user-operator` (description file doesn't exist)
+
+
+## testUserWithNameMoreThan64Chars
+
+**Description:** Verifies that Kafka users with names longer than 64 characters are rejected while correctly named users are accepted.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create Kafka user with correct name (64 characters). | User is successfully created and becomes ready. |
+| 2. | Create SASL user with long name (65 characters). | SASL user is created successfully as SASL users support longer names. |
```

Comment:
```suggestion
| 2. | Create SASL user with long name (65 characters). | SASL user is created successfully, since SASL users support longer names. |
```